cmd/juju/storage/remove.go

```diff
@@ -57,6 +63,11 @@ func (c *removeStorageCommand) Info() *cmd.Info {
 	}
 }
 
+func (c *removeStorageCommand) SetFlags(f *gnuflag.FlagSet) {
+	c.StorageCommandBase.SetFlags(f)
+	f.BoolVar(&c.force, "force", false, "Completely remove the storage and its attachments")
```

Comment:
perhaps reword to match the help text a little closer?
eg Remove storage even if it is currently attached to any units